package.json

```diff
@@ -215,14 +215,17 @@
         "jest": "^24.8.0",
         "npm-run-all": "^4.1.5",
         "opener": "^1.5.1",
+        "pretty-quick": "^3.0.0",
         "source-map-support": "^0.5.12",
         "storybook": "^6.0.13",
         "ts-jest": "^24.0.2",
         "ts-node-dev": "^1.0.0-pre.40",
         "xhr-mock": "^2.5.1"
     },
     "prettier": {
-        "semi": false
+        "semi": false,
+        "trailingComma": "none",
```

Comment:
Might be worth talking about these - I for one am pro-`trailingComma` as it makes diffs cleaner.

But we can also review these at a later point (just like we should at some point go through our eslint rules 😃)

Reply:
Yes I don't have any strong opinions about trailingCommas or other style rules. As long as it can be specified and run on autosave, I'm in favor of it :)